apps/modernization-ui/src/design-system/Pagination/Pagination.spec.tsx

```diff
@@ -111,4 +111,22 @@ describe('When Pagination renders', () => {
 
         expect(toggle).toBeInTheDocument();
     });
+
+    it('should hide the pagination component when the total results are zero', () => {
+        mockTotal = 0;
+
+        const { getByTestId } = render(<Setup />);
+
+        expect(getByTestId('pagination-container-component')).toBeInTheDocument();
```

Comment:
nit: Usually `ByTestId` is a [last resort](https://testing-library.com/docs/queries/bytestid) query.  I think you could do the same using the `queryByText` and checking for "Showing ..."